packages/react-native/package.json

```diff
@@ -34,9 +34,21 @@
   "dependencies": {
     "@noble/ciphers": "^1.2.1",
     "@okto_web3/core-js-sdk": "workspace:*",
+    "@react-navigation/native": "^7.1.6",
+    "@react-navigation/native-stack": "^7.3.10",
+    "@react-navigation/stack": "^7.2.10",
+    "@types/react-native": "^0.73.0",
     "@types/react-native-get-random-values": "^1.8.2",
+    "@types/react-navigation": "^3.4.0",
+    "@types/uuid": "^10.0.0",
+    "expo-clipboard": "^7.0.1",
+    "expo-web-browser": "^14.0.2",
+    "react-native": "^0.79.0",
     "react-native-get-random-values": "^1.11.0",
-    "react-native-mmkv": "^3.2.0"
+    "react-native-mmkv": "^3.2.0",
+    "react-native-safe-area-context": "^5.3.0",
+    "react-native-webview": "^13.6.4",
```

Comment:
@karansinglacoindcx please make RN and expo related dependencies as peerDependencies